GitTfs/Core/IGitTfsRemote.cs

```diff
@@ -38,7 +38,6 @@ public interface IGitTfsRemote
         string TfsUsername { get; set; }
         string TfsPassword { get; set; }
         IGitRepository Repository { get; set; }
-        [Obsolete("Make this go away")]
```

Comment:
@spraints Since I contribute to git-tfs, I have always seen this warning!!!
I have always found it annoying because it add a lot of warning during the build :cry:
I thought sometimes about it but never really understand why you wanted to "make this go away" and what could be done to make this better...
Could you explain me what you wanted to do with that?
I have just decided to remove it because it was here for too long (4 years) to discuss about it here (I think we can live with how git-tfs is now!) but perhaps we could do what you imagined at the beginning... or just convert this "Obsolete" in "TODO"...

Tell me...